pkg/repo/migrations/base_test.go

```diff
@@ -7,13 +7,14 @@ import (
 	"os"
 	"path/filepath"
 
-	"github.com/bacalhau-project/bacalhau/pkg/repo"
 	"github.com/spf13/viper"
 	"github.com/stretchr/testify/suite"
+
+	"github.com/bacalhau-project/bacalhau/pkg/repo"
 )
 
 const (
-	expectedRepoVersion = repo.RepoVersion3
+	expectedRepoVersion = repo.Version3
 )
```

Comment:
should this be version 4?

Reply:
I don't think so, this variable is only used by the v2_3 migration test suite